app/helpers/api_resources_helper.rb

```diff
@@ -1,12 +1,16 @@
 module ApiResourcesHelper
-  def show_file(file)
+  def show_file(file, label)
     return unless file.attached?
 
-    if file.content_type.include?('image')
-      image_tag rails_blob_url(file, subdomain: Apartment::Tenant.current), height: 200, width: 200
-    else
-      link_to file.filename.to_s,  rails_blob_url(file, subdomain: Apartment::Tenant.current), target: '_blank'
-    end
+    file_url = rails_blob_url(file, subdomain: Apartment::Tenant.current)
+    output = ''
+    output << <<-HTML
+    #{image_tag (file.content_type.include?('image') ? file_url : ''), height: 200, width: 200, id: "#{label.parameterize.underscore}_preview_img", style: "display: #{file.content_type.include?('image') ? 'block' : 'none'};"}
```

Comment:
@alis-khadka do we need to restrict the size of the preview? Can we show a responsive preview that makes better use of the space on the page? Please see below for an area marked in blue that is all space we can use for non-primitive property blocks and their previews

<img width="1728" alt="Screen Shot 2022-08-18 at 8 13 47 AM" src="https://user-images.githubusercontent.com/35935196/185392183-a0922b99-5e26-41a6-895c-df179c5b8580.png">

Reply:
@donrestarone I have given max-height to the preview section. Let me know if its okay.

https://user-images.githubusercontent.com/25191509/185544841-a9590b58-c901-41ca-974e-45998209d5df.mp4